website/docs/expansion.md

```diff
@@ -1,97 +1,41 @@
-the Istio example, `Assign` and `ModifySet` mutators could be configured to
-mimic Istio sidecar injection. For further details on mutating mock resources
-see the [Match Source](#match-source) section below, or to see a working example,
-see the [Example](#example) section.
-
-Any resources configured for expansion will be expanded by both the validating
-webhook and
-[Audit](audit.md). This
-feature will only be enabled if a user creates a `ExpansionTemplate` that
-targets some resource that exists on the cluster.
-
-Note that the accuracy of enforcement depends on how well the mock resource
-resembles the real thing. Mutations can help with this, but 100% accuracy is
-impossible because not all fields can be predicted. For instance, deployments
-create pods with random names. Inaccurate mocks may lead to over or under
-enforcement. In the case of under enforcement, the resultant pod should still be
-rejected. Finally, non-state-based policies (those that rely on transient
-metadata such as requesting user or time of creation) cannot be enforced
-accurately. This is because such metadata would necessarily be different when
-creating the resultant resource. For example, a deployment is created using the
-requesting user's account, but the pod creation request comes from the service
-account of the deployment controller.
-
-If, for any reason, you want to exempt expanded resources from a specific
-constraint, look at the [Match Source](#match-source) section below.
-
-## Configuring Expansion
+## Motivation
 
-Expansion behavior is configured through the `ExpansionTemplate` custom
-resource. Optionally, users can create `Mutation` custom resources to customize
-how resources are expanded. Mutators with the `source: Generated` field will
-only be applied when expanding workload resources, and will not mutate real
-resources on the cluster. If the `source` field is not set, the `Mutation` will
-apply to both expanded resources and real resources on the cluster.
+A workload resource is a resource that creates other resources, such as a
+[Deployment](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/) or [Job](https://kubernetes.io/docs/concepts/workloads/controllers/job/). Gatekeeper can be configured to reject workload resources
+that create a resource that violates a constraint. 
 
-Users can test their expansion configuration using the
-[`gator expand` CLI](gator.md)
-.
+## `ExpansionTemplate`s explained
 
-#### ExpansionTemplate
+An `ExpansionTemplate` is a custom resource that Gatekeeper will use to simulate a workload resource expansion. Gatekeeper will create temporary, fake resources and validate the constraints against them. We refer to these resources that Gatekeeper creates for validation purposes as `expanded resources`, but  the terms `mock resources`, `simulated resources`, `resulting resources`, and `resultant resources` all refer to the same notion.
```

Comment:
I wonder if we can limit our use of these different adjectives (expanded, mock, resultant, ect) so that we don't have to explain ourselves. 

We are basically (if not literally) using them all interchangeably. 

Perhaps we can just use the terminology "expanded resources", explain it here like you did, and replace all occurrences of its synonyms?

Reply:
Same thing for verbs.

Reply:
Regarding adjectives and descriptors:

- I replaced everything w `expanded` or reworded things to have `expanded` EXCEPT FOR when we actually define them. 


Regarding verbs:

- I think all verbs other than `expand` have been taken out EXCEPT FOR when we define what it means to expand a resource.